### PR TITLE
feat/comment system

### PR DESCRIPTION
**Pull request by mayingkei**
added post route for creating comment and push to restaurant document

### CODE REVIEW

File: server/models/restaurant.js [hunk 1]
```diff
@@ -17,4 +18,7 @@ const restaurantSchema = Schema({
   comment: [commentSchema],
 });
 
-module.exports = mongoose.model("Restaurant", restaurantSchema);
+module.exports = {
```
Comment (IamMrandrew) [inline server/models/restaurant.js:21]: U can separate Restaurants and Comments model into two files if u want

**Review by IamMrandrew — APPROVED** [review]
Looks ok, didnt actually test

File: server/models/restaurant.js (hunk 1 above)
Comment (mayingkei) [reply server/models/restaurant.js:21]: sure

Comment (mayingkei) [reply server/models/restaurant.js:21]: dou hai suen, only one comment route seems weird

Comment (mayingkei) [reply server/models/restaurant.js:21]: i tried all with postman, so I merge ok?

**Pull request merged**